src/i18n/strings/en_EN.json

```diff
@@ -1845,15 +1845,13 @@
     "You will not be able to undo this change as you are demoting yourself, if you are the last privileged user in the space it will be impossible to regain privileges.": "You will not be able to undo this change as you are demoting yourself, if you are the last privileged user in the space it will be impossible to regain privileges.",
     "You will not be able to undo this change as you are demoting yourself, if you are the last privileged user in the room it will be impossible to regain privileges.": "You will not be able to undo this change as you are demoting yourself, if you are the last privileged user in the room it will be impossible to regain privileges.",
     "Demote": "Demote",
-    "Disinvite this user from %(spaceName)s?": "Disinvite this user from %(spaceName)s?",
-    "Kick this user from %(spaceName)s?": "Kick this user from %(spaceName)s?",
-    "Disinvite this user?": "Disinvite this user?",
-    "Kick this user?": "Kick this user?",
     "Disinvite": "Disinvite",
     "Kick": "Kick",
+    "Disinvite from %(roomName)s": "Disinvite from %(roomName)s",
+    "Kick from %(roomName)s": "Kick from %(roomName)s",
     "Kick them from everything I'm able to": "Kick them from everything I'm able to",
     "Kick them from specific things I'm able to": "Kick them from specific things I'm able to",
-    "If you're not an admin of a room or space in <SpaceName/>, they'll still be able to access it after you kick them.": "If you're not an admin of a room or space in <SpaceName/>, they'll still be able to access it after you kick them.",
+    "If you're not an admin of a room or space in <SpaceName/>, they'll still be able to access whatever you're not an admin of.": "If you're not an admin of a room or space in <SpaceName/>, they'll still be able to access whatever you're not an admin of.",
```

Comment:
Can we change this to be the same as other changed copy?:
   They'll still be able to access whatever you're not an admin of.